src/controllers/playback/video/index.js

```diff
@@ -210,7 +210,18 @@ import { setBackdropTransparency, TRANSPARENCY_LEVEL } from '../../../components
                 itemName = parentName || '';
             }
 
-            LibraryMenu.setTitle(itemName);
+            // Display the item with its premiere date if it has one
+            let title = itemName;
+            if ('PremiereDate' in item) {
```

Comment:
Check not only the presence, but also the value.
```suggestion
            if (item.PremiereDate) {
```